test/fixtures/poro.rb

```diff
@@ -42,6 +41,14 @@ def method_missing(meth, *args)
       super
     end
   end
+
+  def cache_key_with_digest
+    "#{cache_key}/#{FILE_DIGEST}"
+  end
+
+  def updated_at
+    @attributes[:updated_at] ||= DateTime.now.to_time
```

Comment:
I didn't change it in this PR. :) OT: In 1.8 and 1.9 I think DateTime was much slower than Time.  Now, I think avoiding DateTime is mostly about it being the wrong tool and being weird with timezones sometimes...